numba/core/interpreter.py

```diff
@@ -1703,7 +1725,7 @@ def _insert_outgoing_phis(self):
                 val = self.get(varname)
             except ir.NotDefinedError:
                 # Hack to make sure exception variables are defined
-                assert PYVERSION == (3, 11), "unexpected missing definition"
+                assert PYVERSION >= (3, 11), "unexpected missing definition"
```

Comment:
Should this use the "explicit" pattern?

Reply:
Yes, done in 0d3d988f9b94a81ea05fff9d5c82988a382d6ac2